systemtest/src/main/java/io/strimzi/systemtest/kafkaclients/internalClients/admin/AdminClient.java

```diff
@@ -16,6 +19,11 @@ public class AdminClient {
     private final String namespaceName;
     private final String podName;
     private final static String CMD = "admin-client";
+    private final static ObjectMapper MAPPER = new ObjectMapper();
+
+    static {
+        MAPPER.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
+    }
```

Comment:
because this method can be called several times and i did not want to specify it repeatedly; 
also this way it can be configured here for all usages (including future extensions) once only, without need for introducing new methods WDYT ?

Reply:
well when you will need it in future, I guess you can add it? We had more things like this, which were prepared for the future, but actually they were never used and after few months/years they were removed. So in case that you will need it in the future, you can do it that way.

Reply:
yeah i see i was mainly focusing on fact that given command (`describe`) can be called several times and we do no need to set that property several times. But Yeah overall for now setting it this way is probably overkill.

Reply:
From what I saw I think that it wasn't needed, as you anyway have the whole record class for that result, so there will be no unknown properties I suppose. Anyway, thanks